### PR TITLE
Remove unused code in test/e2e/{upgrades,storage,network}

### PR DESCRIPTION
**Pull request by chymy**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
remove unused code in test/e2e/{upgrades,storage,network}, xref: #99243 #99120
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Comment by chymy** [comment]
/assign @jingxu97@erictune
Thanks.

File: test/e2e/storage/csi_mock_volume.go [hunk 1]
```diff
@@ -1631,7 +1631,6 @@ func podRunning(ctx context.Context, c clientset.Interface, podName, namespace s
 }
 
 const (
-	podStartTimeout                = 5 * time.Minute
```
Comment (oomichi) [inline test/e2e/storage/csi_mock_volume.go:1634]: This has been already deleted since 127b675a15bdbb07d57eba24fa0652b7b7b41253

**Review by oomichi — COMMENTED** [review]
A single place has been removed already.
Let's try with current one to see what happens.

/lgtm
/approve

**Pull request merged**